addOns/retire/src/main/java/org/zaproxy/addon/retire/RetireScanRule.java

```diff
@@ -122,14 +124,18 @@ private String getDetails(String key, Map<String, Set<String>> info) {
         }
         StringBuilder sb = new StringBuilder();
         for (String item : info.get(key)) {
+            if (key.equals(Result.CVE)) {
+                alertTags.put(item, "");
+            }
             sb.append(item).append('\n');
         }
         return sb.toString();
     }
 
     @Override
     public Map<String, String> getAlertTags() {
-        return ALERT_TAGS;
+        alertTags.putAll(ALERT_TAGS);
+        return alertTags;
```

Comment:
I don't like this handling but PluginPassiveScanner.AlertBuilder.addTag doesn't seem to be exposed.

https://github.com/zaproxy/zaproxy/blob/260c4a8643015d7763aa152858d1bdee4da98a4e/zap/src/main/java/org/zaproxy/zap/extension/pscan/PluginPassiveScanner.java#L408-L416

![image](https://user-images.githubusercontent.com/7570458/168924094-f4b90aa3-77dc-4cd9-a0a1-6ee8e09ce26e.png)

Reply:
It can use `setTags`.

Reply:
I guess I expected `setTags` to replace not add.

Reply:
Ya that doesn't work. Using `setTags` overrides the "defaults" returned by getTags.

Reply:
Right, you have to set all that you want.

Reply:
In hindsight `setTags` should have been called `setAlertTags` for clarity. `addTag` adds the tag on the message, not alert tags.

Reply:
That method is part of the Alert builder though.

Reply:
Agreed, and that's what I thought at the time as well, but we're using setAlertTags everywhere else too. I don't mind it that much though.

Reply:
Okay. I'll add this to the status doc for discussion Friday. Let's not let it block this PR.

Reply:
Still better to use `setTags`.